scripts/aws/cfn/microservice.yml

```diff
@@ -477,6 +482,29 @@ Resources:
         CloudwatchLogKinesisStreamArn: !Ref LogsKinesisSourceStreamArn
         OpenApiBucketName: !Ref MicroserviceBucketName
         OpenApiBucketKey: !Sub ${MicroserviceBucketBaseKey}/docs/openapi/aws/api-bff-WEB-aws.yaml
+  
+  BffMicroservicePublicWebApiWaf:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: !Sub "${TemplateBucketBaseUrl}/fragments/api-gw-waf-acl.yaml"
+      Parameters:
+        WAFName: !Sub '${ProjectName}-bff-web'
+        IntendedUsage: WEB
+        APIGatewayARNs: !GetAtt BffMicroservicePublicWebAPI.Outputs.APIGatewayARN
+        Limit: !Ref WebWafLimit
+        AlarmSNSTopicArn: !Ref AlarmSNSTopicArn
+        EnhancedWebSecurityEnabled: !Ref EnhancedWebSecurityEnabled
+  
+  BffRecapitiThrottling:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: !Sub "${TemplateBucketBaseUrl}/fragments/applicative-throttling.yaml"
+      Parameters:
+        ApplicationLoadBalancerArn: !Ref ApplicationLoadBalancerArn
+        ThrottledPathsRegexp: '(/bff/v1/addresses/.*)'
+        ThrottlingGroupString: 'recapiti-by-cx-id'
+        MaxRequestPerThrottlingPeriod: 100 # to set to 25 after resolution of aws cloudformation bug: https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/2160  
```

Comment:
@alessioangelotti-pagopa conviene portarti fuori in un parametro cloudformation questo valore (e anche il successivo)

Reply:
Aggiunti parametri BffMaxRequestPerThrottlingPeriod e BffThrottlingPeriodSeconds